flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -445,4 +500,36 @@ protected TableImpl createTable(QueryOperation tableOperation) {
 			operationTreeBuilder,
 			functionCatalog);
 	}
+
+	/**
+	 * Create a result table to show the sql statement execution status, not the data set.
+	 * E.g. A create table DDL statement may returns a table with one column of content:
+	 * "create table success."
+	 * @param result The execution result to show(or for query).
+	 */
+	private TableImpl createResultTable(String result) {
+		QueryOperation queryOperation = new QueryOperation() {
+			@Override
+			public TableSchema getTableSchema() {
+				return TableSchema.builder()
+					.field("RESULT", DataTypes.VARCHAR(Integer.MAX_VALUE))
+					.build();
+			}
+
+			@Override
+			public List<QueryOperation> getChildren() {
+				return new ArrayList<>();
+			}
+
+			@Override
+			public String asSummaryString() {
+				return "Query result operation: " + result;
+			}
+		};
+		return TableImpl.createTable(
```

Comment:
I'm not sure this is a valid way to create a workable table...

Reply:
Returns null for insert and create table now